### ISSUE TITLE
Documentation??

### ISSUE THREAD

**Issue by BugWizard** [CoolMineman/BitsAndChisels#131]
I understand the recipes and all but how come there's actually no documentation on how to use these things?? 
for example could I undo a block I chisel? how does the blueprint work and so on. I feel like there should be some clarification about this.

**Comment by BugWizard** [comment]
Zzz

**Comment by annrandall** [comment]
I agree. I figured out a bunch of stuff by trial and error. For instance, 
(1) the wrench enables you to pivot a chiseled block. Whichever surface you "use" the wrench on is the pivot point.
(2) chiseled blocks can be "mined" and placed elsewhere, but it takes extra mining.
(3) to turn a stack of bits back into blocks, make a pattern for a block that's missing just one bit. When you make a nearly-complete block then add the missing bit, the block will turn back into a normal block.

I considered making a video but didn't have the software I used to use for such and never looked into using something else. Oh and btw, here's some work I did with this mod.
![temple-dome](https://user-images.githubusercontent.com/27778074/185673930-8286ed40-328f-4f42-9282-0714609a412f.jpg)

**Comment by Melechtna** [comment]
I did it once or twice by accident, how do you change the SIZE of what you're placing. I'd like to place the block chunks I'm taking out, rather than taking out chunks and placing dots?
